src/checkpoint/checkpoint.h

```diff
@@ -30,16 +33,11 @@ struct __wt_ckpt_session {
     uint64_t current_sec;
 };
 
-struct __wt_ckpt_connection {
-    WT_SESSION_IMPL *session;       /* Checkpoint thread session */
-    wt_thread_t tid;                /* Checkpoint thread */
-    bool tid_set;                   /* Checkpoint thread set */
-    WT_CONDVAR *cond;               /* Checkpoint wait mutex */
-    wt_shared uint64_t most_recent; /* Clock value of most recent checkpoint */
-#define WT_CKPT_LOGSIZE(conn) (__wt_atomic_loadi64(&(conn)->ckpt.logsize) != 0)
-    wt_shared wt_off_t logsize; /* Checkpoint log size period */
-    bool signalled;             /* Checkpoint signalled */
-
+/*
+ * WT_CKPT_HANDLE --
+ *     Checkpoint information related to handles.
```

Comment:
Since the structure isn't actually handle itself but information about them, maybe call it `WT_CKPT_HANDLE_INFO`?

Reply:
Thank you for the suggestion! Please check [this comment](https://github.com/wiredtiger/wiredtiger/pull/11347/files#r1890695567).